src/modules/user/apiKeys/reducer.ts

```diff
@@ -42,13 +46,16 @@ export const apiKeysReducer = (state = initialApiKeysState, action: ApiKeysActio
                 ...state,
                 apiKeys: sliceArray(action.payload.apiKeys, defaultStorageLimit()),
                 dataLoaded: true,
+                error: undefined,
                 pageIndex: action.payload.pageIndex,
                 nextPageExists: action.payload.nextPageExists,
             };
         case API_KEY_CREATE:
             return {
                 ...state,
                 apiKeys: state.apiKeys.concat(action.payload),
+                modal: {...state.modal, success: true},
+                error: undefined,
```

Comment:
You can reset error value in the FETCH actions
You shouldn't set undefined to it if it's already undefined